modules/core/src/main/java/org/apache/synapse/endpoints/WSDLEndpoint.java

```diff
@@ -77,8 +83,14 @@ public String getWsdlURI() {
         return wsdlURI;
     }
 
+    public String getOriginalWsdlURI() {
+        return this.originalWsdlURI;
+    }
+
     public void setWsdlURI(String wsdlURI) {
-        this.wsdlURI = wsdlURI;
+        this.originalWsdlURI = wsdlURI;
+        String fetchedURI = ConfigUtils.fetchEnvironmentVariables(wsdlURI);
```

Comment:
same comment as AddressEndpoint. fetchedURI local variable is not required.

Reply:
Resolved https://github.com/apache/synapse/pull/55/commits/61fb9e4da909b00d6da8dfe9fb57a3c0a5892793